language/go/testdata/embedsrcs/BUILD.want

```diff
@@ -11,6 +11,7 @@ go_library(
         "m_static.txt",
         "n_/static.txt",
         "o_dir/_hidden.txt",
+        "p_dir/build.old/static.txt",
```

Comment:
Using `build.old` here as the BUILD file name in the tests is to `build.old` here: https://github.com/bazelbuild/bazel-gazelle/blob/f82a89d73ddb8a9ba32980159fbfe26a3a916148/language/go/generate_test.go#L69